src/en/guide/deployment/configuration.md

```diff
@@ -153,30 +153,31 @@ app.update_config(MyConfig())
-| WEBSOCKET_READ_LIMIT      | 2^16              | High-water limit of the buffer for incoming bytes                           |
-| WEBSOCKET_WRITE_LIMIT     | 2^16              | High-water limit of the buffer for outgoing bytes                           |
+| **Variable**              | **Default**           | **Description**                                                                     |
+|---------------------------|-----------------------|-------------------------------------------------------------------------------------|
+| ACCESS_LOG                | True                  | Disable or enable access log                                                        |
+| FALLBACK_ERROR_FORMAT     | html                  | Format of error response if an exception is not caught and handled                  |
+| FORWARDED_FOR_HEADER      | X-Forwarded-For       | The name of "X-Forwarded-For" HTTP header that contains client and proxy ip         |
+| FORWARDED_SECRET          | None                  | Used to securely identify a specific proxy server (see below)                       |
+| GRACEFUL_SHUTDOWN_TIMEOUT | 15.0                  | How long to wait to force close non-idle connection (sec)                           |
+| KEEP_ALIVE                | True                  | Disables keep-alive when False                                                      |
+| KEEP_ALIVE_TIMEOUT        | 5                     | How long to hold a TCP connection open (sec)                                        |
+| PROXIES_COUNT             | None                  | The number of proxy servers in front of the app (e.g. nginx; see below)             |
+| REAL_IP_HEADER            | None                  | The name of "X-Real-IP" HTTP header that contains real client ip                    |
+| REGISTER                  | True                  | Whether the app registry should be enabled                                          |
+| REQUEST_BUFFER_QUEUE_SIZE | 100                   | Request streaming buffer queue size                                                 |
+| REQUEST_BUFFER_SIZE       | 65536                 | Request buffer size before request is paused, default is 64 Kib                     |
+| REQUEST_ID_HEADER         | X-Request-ID          | The name of "X-Request-ID" HTTP header that contains request/correlation ID         |
+| REQUEST_MAX_SIZE          | 100000000             | How big a request may be (bytes), default is 100 megabytes                          |
+| REQUEST_TIMEOUT           | 60                    | How long a request can take to arrive (sec)                                         |
+| RESPONSE_TIMEOUT          | 60                    | How long a response can take to process (sec)                                       |
+| USE_UVLOOP                | True (except Windows) | Whether to override the loop policy to use `uvloop`. Supported only with `app.run`. |
```

Comment:
The default is actually `_default`, but since it's not a public value I've set it to `True` in here. Is that ok or should I put `_default` and leave it like that?

Reply:
You are correct that the "default" is `True` since that is what the user cares about. The sentinel is not something that needs to be explained.